migration-from-4x.md

```diff
@@ -557,19 +557,30 @@ Changes between TinyMCE 4.x and TinyMCE 5.0:
 
 ### Other removed component configuration settings
 
-Some UI configurations in TinyMCE 5.0 have changed due to an updated UI engine. All inline style configurations have been removed in TinyMCE 5.0 in favor of modern CSS.  This affects all TinyMCE 4.x UI component configurations.
+All inline style configurations have been removed in TinyMCE 5.0 in favor of modern CSS.  This affects all TinyMCE 4.x UI component configurations.
 The table below shows UI configurations that have been removed. They are generally related to inline CSS settings.
 
-| **Old settings** | **Description** | **Alternative** |
```

Comment:
These still seems relevant, as theses were properties that you used to be able to set inside the `editor.windowManager.open` call?

Reply:
replaced table with a list